arma3/@task_force_radio/addons/task_force_radio/functions/common.sqf

```diff
@@ -21,7 +21,7 @@ TF_CODE_OFFSET = TF_SW_STEREO_OFFSET + 1;
 TF_ADDITIONAL_CHANNEL_OFFSET = TF_CODE_OFFSET + 1;
 TF_ADDITIONAL_STEREO_OFFSET = TF_ADDITIONAL_CHANNEL_OFFSET + 1;
 
-TF_LR_SPEAKER_OFFSET = 7;
+TF_LR_SPEAKER_OFFSET = 8;
```

Comment:
I believe that was done because the player UID isn't needed at all in the LR radio stuff. Due to them being vehicles. But it was done to get the arrays to be the same length at the very least. I think.